src/components/list/demoListControls/index.html

```diff
@@ -0,0 +1,29 @@
+<md-list ng-controller="ListCtrl">
+  <md-subheader class="md-no-sticky">Single Action Checkboxes</md-subheader>
+  <md-item ng-repeat="topping in toppings">
+    <p> {{ topping.name }} </p>
+    <md-checkbox class="md-secondary" ng-model="topping.wanted" aria-label="{{ topping.name }}">
+  </md-item>
+  <md-subheader class="md-no-sticky">Clickable Items with Secondary Controls</md-subheader>
+  <md-item ng-click="navigateTo(setting.extraScreen)" ng-repeat="setting in settings">
+    <md-icon md-svg-icon="{{setting.icon}}"></md-icon>
+    <p> {{ setting.name }} </p>
+    <md-switch class="md-secondary" ng-model="setting.enabled" aria-label="{{ setting.name }}">
+  </md-item>
+  <md-item ng-click="navigateTo('data usage')">
+    <md-icon md-svg-icon="cached"></md-icon>
+    <p>Data Usage</p>
+  </md-item>
+  <md-subheader class="md-no-sticky">Checkbox with Secondary Action</md-subheader>
+  <md-item ng-repeat="message in messages">
+    <md-checkbox ng-model="message.selected" aria-label="Select"></md-checkbox>
+    <p>{{message.title}}</p>
+    <md-icon class="md-secondary" ng-click="doSecondaryAction()", md-svg-icon="communication:message"></md-icon>
```

Comment:
This item needs an `aria-label` if it has `ng-click` on it. Without `aria-label` present, `md-icon` gets `aria-hidden="true"` because it assumes you meant it was for presentation only. So we'll have to enforce it somehow.